test/fixtures/themes/theme-with-block-partials/index.hbs

```diff
@@ -0,0 +1,3 @@
+{{#> block}}
+{{nope "string"}}
+{{/block}}
```

Comment:
Some of the test theme files can be removed i guess e.g. `logo.new.hbs` or `subfolder/test.hbs`